.pre-commit-config.yaml

```diff
@@ -0,0 +1,9 @@
+# Configuration file for pre-commit (https://pre-commit.com/).
+# Please run `pre-commit run --all-files` when adding or changing entries.
+
+repos:
+  - repo: https://github.com/pre-commit/mirrors-prettier
```

Comment:
Do you prefer installing from https://github.com/pre-commit/mirrors-prettier or [using the Nix package](https://gitlab.com/engmark/root/-/blob/8486252da2aedaee084fae1c6366f7217b35d6e4/.pre-commit-config.yaml#L46-52)?

Reply:
I'm completely unfamiliar with that, so I'd have to look into it. Unfortunately devenv isn't even in nixpkgs, so this will require a fair bit more setup. If someone else is up for converting this to devenv, feel free!